packages/scene-composer/src/augmentations/components/three-fiber/anchor/AnchorWidget.tsx

```diff
@@ -124,11 +128,48 @@ export function AsyncLoadedAnchorWidget({
       WarningIconSvgString,
       ErrorIconSvgString,
       VideoIconSvgString,
+      CustomIconSvgString,
     ];
     return iconStrings.map((iconString, index) => {
+      if (keys[index] === 'Custom') {
+        const parser = new DOMParser();
+        const svgDocument = parser.parseFromString(iconString, 'image/svg+xml');
+        const svgRoot = svgDocument.documentElement;
+
+        const replaceFillAttribute = (element: Element) => {
+          const tagName = element.tagName.toLowerCase();
+          if (tagName === 'ellipse') {
+            if (chosenColor === undefined) {
+              return;
+            } else {
```

Comment:
these `if`s can be reorganized to simplify
```
if (chosenColor === undefined) {
  return;
}

if (tagName === 'ellipse') {
  .....
}
```

Reply:
I did. next cr.